cli/command/stack/deploy.go

```diff
@@ -30,7 +30,7 @@ func newDeployCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "deploy [OPTIONS] STACK",
-		Aliases: []string{"up"},
+		Aliases: []string{"up", "docker deploy"},
```

Comment:
Ah, clear. I'm ok with merging this as-is, but it would be a nice improvement, so up to you if you want to do it as part of this PR. Otherwise, LGTM